infra/bootstrapper/resource_groups.tf

```diff
@@ -0,0 +1,59 @@
+import {
+  id = "/subscriptions/ec285037-c673-4f58-b594-d7c480da4e8b/resourceGroups/io-p-ioweb-sec-rg"
+  to = azurerm_resource_group.sec_weu
+}
+
+resource "azurerm_resource_group" "sec_weu" {
+  name     = "${local.prefix}-${local.env_short}-ioweb-sec-rg"
+  location = "westeurope"
+
+  tags = local.tags
+}
```

Comment:
I think it's better to move to BE monorepo, since it contains the keyvault with most of BE secrets

Reply:
Are you planning to use two repositories for ioweb then?